<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1240</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1240</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MENU</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon_pack.qrc">
    <normaloff>:/images/network.png</normaloff>:/images/network.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/images/pink-gradient-white-linear-3840x2160-c2-ffb6c1-ffffff-a-15-f-14.png)</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>171</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon_pack.qrc">:/images/Thanksgiving-33-512_151x151.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(164, 0, 0);
	border-radius:17px;
	border:1px solid ;
	display:inline-block;
color:rgb(238, 238, 236);
font-family:Courier New;
font-size:15px;
font-weight:bold;
padding:7px 24px;
text-decoration:none;
text-shadow:0px 1px 0px #c70067;
}
QPushButton:hover{
background:rgb(239, 41, 41);

}
</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>270</y>
      <width>161</width>
      <height>227</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttonlayout">
     <property name="spacing">
      <number>18</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:rgb(0, 0, 0);
	border-radius:17px;
	border:1px solid ;
	display:inline-block;
color:rgb(238, 238, 236);
font-family:Courier New;
font-size:15px;
font-weight:bold;
padding:7px 24px;
text-decoration:none;
text-shadow:0px 1px 0px #c70067;
}
QPushButton:hover{
background:rgb(85, 87, 83);

}
QPushButton:clicked{
background:rgb(85, 87, 83);
}
</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset resource="icon_pack.qrc">
         <normaloff>:/images/graph.png</normaloff>:/images/graph.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:rgb(0, 0, 0);
	border-radius:17px;
	border:1px solid ;
	display:inline-block;
color:rgb(238, 238, 236);
font-family:Courier New;
font-size:15px;
font-weight:bold;
padding:7px 24px;
text-decoration:none;
text-shadow:0px 1px 0px #c70067;
}
QPushButton:hover{
background:rgb(85, 87, 83);

}
QPushButton:clicked{
background:rgb(85, 87, 83);
}</string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
       <property name="icon">
        <iconset resource="icon_pack.qrc">
         <normaloff>:/images/mathematics.png</normaloff>:/images/mathematics.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>23</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:rgb(0, 0, 0);
	border-radius:17px;
	border:1px solid ;
	display:inline-block;
color:rgb(238, 238, 236);
font-family:Courier New;
font-size:15px;
font-weight:bold;
padding:7px 24px;
text-decoration:none;
text-shadow:0px 1px 0px #c70067;
}
QPushButton:hover{
background:rgb(85, 87, 83);

}
QPushButton:clicked{
background:rgb(85, 87, 83);
}
</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
       <property name="icon">
        <iconset resource="icon_pack.qrc">
         <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:rgb(0, 0, 0);
	border-radius:17px;
	border:1px solid ;
	display:inline-block;
color:rgb(238, 238, 236);
font-family:Courier New;
font-size:15px;
font-weight:bold;
padding:7px 24px;
text-decoration:none;
text-shadow:0px 1px 0px #c70067;
}
QPushButton:hover{
background:rgb(85, 87, 83);

}
QPushButton:clicked{
background:rgb(85, 87, 83);
}</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="icon_pack.qrc">
         <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>1051</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="showBox">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true">background:rgb(204, 0, 0);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#eeeeec;&quot;&gt;PENDING TASKS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>121</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icon_pack.qrc">:/images/unnamed_121x111.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>191</width>
         <height>101</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 56pt &quot;DejaVu Sans&quot;;
font-weight:bold;
color:rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">background:rgb(64, 103, 191)</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#eeeeec;&quot;&gt;MUST COMPLETE TASKS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>121</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icon_pack.qrc">:/images/things-already-done-23a776c71d9ec435984843b7c43103f8_121x111.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>191</width>
         <height>101</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 56pt &quot;DejaVu Sans&quot;;
font-weight:bold;
color:rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">background:rgb(115, 210, 22)</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#eeeeec;&quot;&gt;COMPLETED TASKS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>70</y>
         <width>121</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icon_pack.qrc">:/images/-png-_121x111.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>191</width>
         <height>101</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 56pt &quot;DejaVu Sans&quot;;
font-weight:bold;
color:rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>1051</width>
     <height>381</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Liberation Sans&quot;;

QTableView::item:focus{
selection-background:rgb(186, 189, 182)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::CustomDashLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>375</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QGroupBox" name="addbox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>1051</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;DejaVu Sans Mono&quot;;
color:rgb(238, 238, 236);
background:rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>ADD TASK</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;FILL THE TASK DETAILS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;NAME&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;PRIORITY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;TASK STATUS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Set Task Name"/>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(211, 215, 207);
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(211, 215, 207);
color:rgb(0, 0, 0);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>HIGH LOW MEDIUM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(211, 215, 207);
color:rgb(0, 0, 0);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>0-UNFINISHED  1-FINISHED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(14, 111, 249);
	border-radius:17px;
	border:1px solid ;
	display:inline-block;
color:rgb(238, 238, 236);
font-family:Courier New;
font-size:15px;
font-weight:bold;
padding:7px 24px;
text-decoration:none;
text-shadow:0px 1px 0px #c70067;
}
QPushButton:hover{
background:rgb(113, 194, 247);

}
</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon_pack.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>menu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>addbox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
